src/pages/resultsView/querySummary/QuerySummaryUtils.tsx

```diff
@@ -10,13 +10,13 @@ export function getPatientSampleSummary(samples: any[], patients: any[]) {
         const sampleUnits = samples.length === 1 ? 'sample' : 'samples';
```

Comment:
this is a comment about this whole function.   i do realize this is not your code.   but we could take opportunity to improve it.  1) probably this should be a component.     2) we should improve on the typing of the arguments.  anytime we see use of any, we should try to fix it.   in this case, patients looks like it's string[].   for samples, there is an existing type, SampleIdentifiers which i think will work here.

Reply:
sample and patients can be Sample[] or Patient[] respectively as well so I added those along with the SampleIdentifier[] and string[] types